src/routes/github/github-oauth-router.ts

```diff
@@ -208,5 +209,5 @@ const getCloudOrGHESAppClientSecret = async (gitHubAppConfig, jiraHost: string)
 // IMPORTANT: We need to keep the login/callback/middleware functions
 // in the same file as they reference each other
 export const GithubOAuthRouter = Router();
-GithubOAuthRouter.get("/login", GithubOAuthLoginGet);
-GithubOAuthRouter.get(callbackSubPath, GithubOAuthCallbackGet);
+GithubOAuthRouter.get("/login", GithubServerAppMiddleware, jiraSymmetricJwtMiddleware, GithubOAuthLoginGet);
```

Comment:
I think we should swap the order by `jiraSymmetricJwtMiddleware` and `GithubServerAppMiddleware`